actors/builtin/miner/policy.go

```diff
@@ -190,21 +245,28 @@ type VestSpec struct {
 	Quantization abi.ChainEpoch // Maximum precision of vesting table (limits cardinality of table).
 }
 
+// PARAM_SPEC
 var PledgeVestingSpec = VestSpec{
 	InitialDelay: abi.ChainEpoch(180 * builtin.EpochsInDay), // PARAM_FINISH
 	VestPeriod:   abi.ChainEpoch(180 * builtin.EpochsInDay), // PARAM_FINISH
 	StepDuration: abi.ChainEpoch(1 * builtin.EpochsInDay),   // PARAM_FINISH
 	Quantization: 12 * builtin.EpochsInHour,                 // PARAM_FINISH
 }
 
+// PARAM_SPEC
 var RewardVestingSpec = VestSpec{
 	InitialDelay: abi.ChainEpoch(20 * builtin.EpochsInDay),  // PARAM_FINISH
```

Comment:
**Clarification request:** Hey @zixuanzh, Is this a final parameter? I recall that this used to be ~180 days. This means that a miner earns block rewards with a delay of 20 days.

Reply:
these are not final numbers and yes it used to be 180 days of delay

Reply:
Ok, waiting on final param to close this.